apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-step/index.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './styles.scss';
+
+export interface Props {
+	className?: string;
+	onPrevStep?: () => void;
+	onNextStep?: () => void;
```

Comment:
I'm not sure but I think these don't belong here, right?

Reply:
They're not used in this base component but used by the components e.g. `DomainStep`, `PlanStep` & `PrivacyStep` that uses this. For now `LaunchStep` is just a base component to give us more leeway when we need something that's shared across all steps in the future.